internal/rpmostreeclient/rpmostreeclient.go

```diff
@@ -0,0 +1,91 @@
+package rpmostreeclient
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+	"syscall"
+)
+
+// Status summarizes the current worldview of the rpm-ostree daemon.
+// The deployment list is the primary data.
+type Status struct {
+	// Deployments is the list of bootable filesystem trees.
+	Deployments []Deployment
+	// Transaction is the active transaction, if any.
+	Transaction *[]string
+}
+
+// Deployment represents a bootable filesystem tree
+type Deployment struct {
+	ID                      string   `json:"id"`
+	OSName                  string   `json:"osname"`
+	Serial                  int32    `json:"serial"`
+	BaseChecksum            *string  `json:"base-checksum"`
+	Checksum                string   `json:"checksum"`
+	Version                 string   `json:"version"`
+	Timestamp               uint64   `json:"timestamp"`
+	Booted                  bool     `json:"booted"`
+	Staged                  bool     `json:"staged"`
+	LiveReplaced            string   `json:"live-replaced,omitempty"`
+	Origin                  string   `json:"origin"`
+	CustomOrigin            []string `json:"custom-origin"`
+	ContainerImageReference string   `json:"container-image-reference"`
+	RequestedPackages       []string `json:"requested-packages"`
+	RequestedBaseRemovals   []string `json:"requested-base-removals"`
+	Unlocked                *string  `json:"unlocked"`
+}
```

Comment:
@donpenney which of these fields are good for the stateroot list in IBU status? I am guessing id, osname, booted at minimum?

Reply:
@jc-rh Depends on what the intent is. The stateroot itself, ie. rhcos_$version, is the OSName here. The ID would be the stateroot + the deployment commit id and iterator. If it's for determining which release is active (A or B side), then we'd just need the OSName and Booted. The ID likely doesn't provide any use for LCA directly